kibbeh/package.json

```diff
@@ -17,53 +17,53 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@date-io/date-fns": "^1.3.13",
+    "@date-io/date-fns": "^2.10.8",
```

Comment:
this needs to stay at v1 its specified in readme for lib we are using

Reply:
Fixed